osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -236,18 +234,22 @@ public void Download(BeatmapSetInfo beatmapSetInfo, bool noVideo = false)
 
             if (!api.LocalUser.Value.IsSupporter)
             {
-                PostNotification?.Invoke(new SimpleNotification
-                {
-                    Icon = FontAwesome.fa_superpowers,
-                    Text = "You gotta be a supporter to download for now 'yo"
-                });
+                PostNotification?.Invoke(
+                    new Notification(
+                        "You gotta be a supporter to download for now 'yo",
+                        FontAwesome.fa_superpowers
+                    )
+                );
                 return;
             }
 
-            ProgressNotification downloadNotification = new ProgressNotification
-            {
-                Text = $"Downloading {beatmapSetInfo.Metadata.Artist} - {beatmapSetInfo.Metadata.Title}",
-            };
+            ProgressNotification downloadNotification = new ProgressNotification(
+                $"Downloading {beatmapSetInfo.Metadata.Artist} - {beatmapSetInfo.Metadata.Title}"
+            );
+
+            downloadNotification.FollowUpNotifications.Add(
+                new Notification($"Downloading {beatmapSetInfo.Metadata.Artist} - {beatmapSetInfo.Metadata.Title} finished")
+            );

```

Comment:
Inline, basically just go through this PR and inline all of these. Makes it easier to follow imo.